spacy/pipeline/morphologizer.pyx

```diff
@@ -25,6 +25,7 @@ default_model_config = """
 
 [model.tok2vec]
 @architectures = "spacy.Tok2Vec.v1"
+pretrained_vectors = null
```

Comment:
I don't think we need this line. The `pretrained_vectors` argument is passed to the `spacy.HashEmbedCNN.v1` function because that function takes all the arguments to the embedding and encoding layers together, because it builds them both. The `spacy.Tok2Vec.v1` layer takes in the embedding and encoding layers as arguments, so we don't have this config option.